src/state/reducers/orderItemsReducer.js

```diff
@@ -0,0 +1,63 @@
+import { createReducer, createAction } from '@reduxjs/toolkit';
+
+export const addItemToOrders = createAction('ADD_ITEM_TO_ORDERS');
+export const addToPriceTotal = createAction('ADD_TO_PRICE_TOTAL');
+export const resetAllOrders = createAction('RESET_ALL_ORDERS');
+export const updateItemName = createAction('UPDATE_ITEM_NAME');
+export const updateItemId = createAction('UPDATE_ITEM_ID');
+export const updateItemQuantity = createAction('UPDATE_ITEM_QUANTITY');
+export const updateTotalPrice = createAction('UPDATE_TOTAL_PRICE');
+export const updateRestaurantId = createAction('UPDATE_RESTAURANT_ID');
+export const setUserId = createAction('SET_USER_ID');
+export const setGroupId = createAction('SET_GROUP_ID');
+
+const initialState = {
+  orders: [],
+  ordersTotal: 0,
+  currentOrder: {
+    user_id: 0,
+    food: '',
+    quantity: 0,
+    price: 0,
+    date: '',
+    food_id: 0,
+    group_id: 0,
+    restaurant_id: 0,
+    live: true,
+  },
+};
+
+export const orderItemsReducer = createReducer(initialState, (builder) => {
```

Comment:
recommend changing the reducer name to something more general, just like ordersReducer? 

I think because orderItemsReducer having a state called "orders" is a little confusing having a nested state be a larger collection than suggested by the name of the reducer. ("orderItems" sounds more specific than "orders")

While at it, the state "orders" could be more specific. Something like "currentOrders" or "allOrders" (i forget what context this state is used in). That way when we retrieve the state, it would be more readable like
`state => state.orders.currentOrders`

Reply:
Gotcha Gotcha lemme work on that 👍